packages/sdk-codegen-scripts/src/fetchSpec.ts

```diff
@@ -186,27 +186,16 @@ export const login = async (props: ISDKConfigProps) => {
   }
   const url = loginUrl(props)
 
-  try {
-    const response = await sdkOk<any, Error>(
-      xp.request<any, Error>(
-        'POST',
-        url,
-        creds,
-        undefined,
-        undefined,
-        undefined
-      )
-    )
-    const accessToken = await response.access_token
+  const response = await sdkOk<any, Error>(
+    xp.request<any, Error>('POST', url, creds, undefined, undefined, undefined)
+  )
+  const accessToken = await response.access_token
 
-    if (accessToken) {
-      return accessToken
-    } else {
-      log(`Server Response: ${JSON.stringify(response)}`)
-      throw new Error('Access token could not be retrieved.')
-    }
-  } catch (err) {
-    throw err
```

Comment:
😸

Reply:
yeah, I think that was left over from some testing I was doing when first writing that code.